artifactory/utils/buildinfoproperties.go

```diff
@@ -164,19 +164,20 @@ func ReadGradleConfig(path string, useWrapperIfMissingConfig bool) (config *vipe
 	return
 }
 
-func ReadMavenConfig(path string) (config *viper.Viper, err error) {
+func ReadMavenConfig(path string, useWrapperIfMissingConfig bool) (config *viper.Viper, err error) {
 	if path == "" {
-		config = createDefaultMavenConfig()
+		config = createDefaultMavenConfig(useWrapperIfMissingConfig)
 	} else {
 		config, err = ReadConfigFile(path, YAML)
 	}
 	return
 }
 
-func createDefaultMavenConfig() *viper.Viper {
+func createDefaultMavenConfig(useWrapperIfMissingConfig bool) *viper.Viper {
```

Comment:
useWrapperIfMissingConfig --> useWrapper

(this function is unaware of the "if missing in config" logic).

Reply:
Done